### PR TITLE
Add animation duration multiplier configuration

### PR DESCRIPTION
**Pull request by tobrun**
This PR adds the ability to configure the animation duration multiplier used.

### CODE REVIEW

File: plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java [hunk 1]
```diff
@@ -1534,6 +1564,17 @@ private Builder(LocationLayerOptions source) {
       return this;
     }
 
+    /**
+     * Sets the animation duration multiplier.
+     *
+     * @param animationDurationMultiplier the animation duration multiplier
+     * @since 0.9.0
+     */
+    public LocationLayerOptions.Builder animationDurationMultiplier(float animationDurationMultiplier) {
```
Comment (LukasPaczos) [inline plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java]: Could we indicate that this is a tracking animation duration? Either in the method name, javadoc or both works.

File: plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java [hunk 2]
```diff
@@ -748,6 +769,7 @@ public String toString() {
       + "trackingInitialMoveThreshold=" + trackingInitialMoveThreshold + ", "
       + "trackingMultiFingerMoveThreshold=" + trackingMultiFingerMoveThreshold + ", "
       + "layerBelow=" + layerBelow
+      + "animationDurationMultiplier=" +animationDurationMultiplier
```
Comment (LukasPaczos) [inline plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java]: NIT whitespace missing

File: plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java (hunk 1 above)
Comment (tobrun) [reply plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java]: Yes, went with updating all occurrences of the name by pre-preding tracking to it so it conforms to other tracking mode configurations. Thanks for the quick 👀.

**Comment by LukasPaczos** [comment]
Changing the multiplier at this point requires recreating the plugin. If that should be the case, javadoc entry would be great, otherwise, we should reload the multiplier in `PluginAnimatorCoordinator` whenever new options are provided.

**Comment by tobrun** [comment]
Thanks for catching that, in https://github.com/mapbox/mapbox-plugins-android/pull/706/commits/6b0a17642e87d574ecaca3368f83b23bc660a725 I replaced the constructor configuration with a setter. That felt cleaner as recreating the object. Ready for review :)

**Pull request merged**